modules/client/src/test/kotlin/jp/co/soramitsu/iroha2/InstructionsTest.kt

```diff
@@ -333,6 +335,18 @@ class InstructionsTest : IrohaTest<Iroha2Client>(
         assertEquals(asset, assetsByDomainName.first())
     }
 
+    @Test
+    @WithIroha(
+        [DefaultGenesis::class],
+        configs = ["WSV_ACCOUNT_METADATA_LIMITS$IROHA_CONFIG_DELIMITER{\"max_entry_byte_size\": 65536, \"max_len\": 1048576}"]
```

Comment:
how about representing that as a map or tuples, i.e. `WSV_ACCOUNT_METADATA_LIMITS to <>`